mob.go

```diff
@@ -1092,7 +1092,8 @@ func sendRequest(requestBody []byte, requestMethod string, requestUrl string, di
 		switch e.Err.(type) {
 		case x509.UnknownAuthorityError:
 			sayError("The timer.mob.sh SSL certificate is signed by an unknown authority!")
-			sayError("HINT: You can ignore that by adding MOB_TIMER_INSECURE=true to your configuration or environment.")
+			sayFix("HINT: You can ignore that by adding MOB_TIMER_INSECURE=true to your configuration or environment. Or add is command line parameter:",
+				"mob <your command> --timer-insecure")
```

Comment:
Can you reproduce the actual command that was used? Should either be timer or break, and add the timer value to it as well. I would like to have the fix as a copy'n'paste solution for the user without any placeholders. For that, we might need to pass more parameters, but I take that into account.